libraries/entities-renderer/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 set(TARGET_NAME entities-renderer)
-setup_hifi_library(Network Script)
-link_hifi_libraries(shared workload gpu shaders procedural graphics material-networking model-networking script-engine render render-utils image qml ui pointers)
+setup_hifi_library(Network)
+link_hifi_libraries(shared workload gpu shaders procedural graphics material-networking model-networking script-engine render render-utils image qml ui pointers entities)
```

Comment:
Hmm, switching that from a link to an include seems to be working now.  I'd prefer to have as few lines as possible for that, will remove.

Reply:
Ah nevermind, the macos build was failing without entities being linked, leaving this in.

Reply:
yeah I'd expect it to need it, which is why I'm surprised it didn't have it before /shrug

Reply:
I suspect that if I dug into this more it would end up being some script class that I moved out of script-engine into entities, just not worth the bother right now trying to track that down anymore.